lua/lvim/plugin-loader.lua

```diff
@@ -39,6 +39,7 @@ function plugin_loader.init(opts)
 
   if in_headless then
     init_opts.display = nil
+    init_opts.git.clone_timeout = 240
```

Comment:
we can set it the same as `curl`[^1] 

```suggestion
    init_opts.git.clone_timeout = 300
```

[^1]: https://curl.se/libcurl/c/CURLOPT_CONNECTTIMEOUT_MS.html

Reply:
partially reverts https://github.com/LunarVim/LunarVim/pull/2910